### PR TITLE
[Improvement] not print logs to fe.out when fe is running under daemon

### PR DESCRIPTION
**Pull request by yiguolei**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/common/status.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ namespace doris {
 
 // ErrorName, ErrorCode, String Description, Should print stacktrace
 #define APPLY_FOR_ERROR_CODES(M) \
-    M(OLAP_SUCCESS, 0, "", true) \
+    M(OLAP_SUCCESS, 0, "", false) \
```
Comment (wangshuo128) [inline be/src/common/status.h:25]: Why change this?

Comment (morningman) [reply be/src/common/status.h:25]: Because we don't need to print stack trace for `OLAP_SUCCESS`

**Review by morningman — APPROVED** [review]
LGTM

**Comment by Toms1999** [comment]
let me seesee

**Pull request merged**